lib/web_ui/dev/felt

```diff
@@ -45,7 +45,7 @@ then
   echo "Compiling the Dart SDK."
   gclient sync
   $GN --unoptimized --full-dart-sdk
-  ninja -C $HOST_DEBUG_UNOPT_DIR -j 100
+  "$NINJA_PATH" -C "$HOST_DEBUG_UNOPT_DIR"
```

Comment:
Thanks for improving this!

There's another place we use `ninja`: https://github.com/flutter/engine/blob/ca731e6ac3e3fad6f5505bb62a0b8a5c5c9f1cbf/lib/web_ui/dev/build.dart#L88

Reply:
Awesome. Thanks for letting me know! It looks like there's a bunch of parameter passing options etc. and some readme updates; will put together a separate PR for that work!